scholia/app/templates/venue.html

```diff
@@ -39,7 +39,7 @@
 
  prolificAuthorsSparql = `
 # Prolific authors for a specific journal
-SELECT ?count ?author ?authorLabel ?orcid
+SELECT ?count ?author ?authorLabel ?ORCID
```

Comment:
Ok I see that the header in the table on does not uppercase here: https://tools.wmflabs.org/scholia/venue/Q6294930#Prolific%20authors